### PR TITLE
Migrate to TensorFlow v2

### PR DESCRIPTION
**Pull request by chezou**
TensorFlow v2 is based on Keras and interface is totally different from v1. I rewrite the sentiment analysis codes with TF v2.

### CODE REVIEW

File: machine-learning-box/sentiment-analysis/py_scripts/data.py [hunk 1]
```diff
@@ -6,6 +6,7 @@
 
 os.system(f"{sys.executable} -m pip install -U pytd==0.8.0 td-client")
```
Comment (takuti) [inline machine-learning-box/sentiment-analysis/py_scripts/data.py]: Just noticed `import sys` is missing before this line.

File: machine-learning-box/sentiment-analysis/py_scripts/predict.py [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+    )
+    model.summary()
+
+    result = model.predict(df["sentence"].values)
+
+    print("upload prediction results")
+    table = "test_predicted_polarities"
+    df["predicted_polarity"] = np.where(result > 0.5, 1, 0)
+    client.load_table_from_dataframe(
+        df["rowid", "predicted_polarity"], f"{database}.{table}", if_exists="overwrite"
```
[the remark this answers is not in the document]
Comment (chezou) [reply machine-learning-box/sentiment-analysis/py_scripts/predict.py]: fixed 
0471d86

File: machine-learning-box/sentiment-analysis/py_scripts/sentiment.py [hunk 3]
```diff
@@ -0,0 +1,116 @@
...
+    # Original code is published at official document of TensorFlow under Apache License Version 2.0
+    # https://www.tensorflow.org/hub/tutorials/text_classification_with_tf_hub
+
+    os.system(
+        f"{sys.executable} -m pip install -U tensorflow==2.0.0 tensorflow_hub==0.7.0"
+    )
+
+    import tensorflow as tf
+    import tensorflow_hub as hub
+    import pytd.pandas_td as td
```
[the remark this answers is not in the document]
Comment (chezou) [reply machine-learning-box/sentiment-analysis/py_scripts/sentiment.py]: Rewrote it  0471d86

File: machine-learning-box/sentiment-analysis/py_scripts/data.py (hunk 1 above)
Comment (chezou) [reply machine-learning-box/sentiment-analysis/py_scripts/data.py]: oh my bad... 
0471d86

File: machine-learning-box/sentiment-analysis/py_scripts/predict.py [hunk 4]
```diff
@@ -0,0 +1,59 @@
...
+            from
+                {input_table}
+        """
+    )
+    df = pd.DataFrame(**query_result)
+
+    print("load model")
+    model_file = "keras_model.h5"
+
+    # Download the TensorFlow model to S3
```
[the remark this answers is not in the document]
Comment (chezou) [reply machine-learning-box/sentiment-analysis/py_scripts/predict.py]: good catch 0471d86

**Pull request merged**